extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -93,13 +106,28 @@ public void initialize() {
                 updateStatus(statusInfo.getStatus(), statusInfo.getStatusDetail(), statusInfo.getDescription());
                 FullLight fullLight = getLight();
                 if (fullLight != null) {
+                    String modelId = fullLight.getModelID().replaceAll(NORMALIZE_ID_REGEX, "_");
+                    updateProperty(Thing.PROPERTY_MODEL_ID, modelId);
```

Comment:
As this discussion is taking place in the HueLightHandler, this is exactly what @kaikreuzer meant: updating the firmware version property inside the handler (see line below). But I could hardly imagine how to do this otherwise. I think, #588 needs to be implemented soon to finish this discussion here. :-)